packages/table/src/headers/_headers.scss

```diff
@@ -83,25 +84,45 @@
     height: $interaction-bar-height;
     text-align: center;
 
-    > span {
+    > [class*="pt-icon"] {
```

Comment:
This selector seems overly complicated, why not just style `.pt-icon` or `.pt-icon-standard` (whatever the sizing class is)? same below